src/main/java/uk/gov/hmcts/ccd/domain/service/getevents/ClassifiedGetEventsOperation.java

```diff
@@ -23,7 +24,7 @@
 
     public ClassifiedGetEventsOperation(@Qualifier("default") GetEventsOperation getEventsOperation,
                                         SecurityClassificationService classificationService,
-                                        @Qualifier(CreatorGetCaseOperation.QUALIFIER)
+                                        @Qualifier(AccessControlledGetCaseOperation.QUALIFIER)
```

Comment:
The various controllers are injected with `AuthorisedGetEventsOperation`, so I think we should replaced that with AccessControlledGetCaseOperation rather than this class